.github/workflows/lint.yml

```diff
@@ -150,22 +150,25 @@ jobs:
         env:
           HEAD_SHA: ${{ github.event.pull_request.head.sha }}
         run: |
-          mkdir flake8-output
+          mkdir -p flake8-output
```

Comment:
Why is the `-p` necessary here?

Reply:
it silents the "folder already exist" error i think?

Reply:
Yup, but it's not necessary for this anymore since I moved some of the steps around

Reply:
@driazati In that case could you change it back to how it was before (just to keep the diff clean)?